lib/avahi.js

```diff
@@ -18,10 +18,12 @@ function supportsInterfaceIndexLocalOnly() {
           host, port, txtRec, cb, ctx);
   } catch (ex) {
     if (ex.errorCode === dns_sd.kDNSServiceErr_Unsupported) {
+      dns_sd.DNSServiceRefDeallocate(sr)
```

Comment:
Sorry to nitpick, but could you terminate the statements with semicolons like elsewhere in the codebase?